public/scripts/chat/chat.js

```diff
@@ -218,14 +218,14 @@ function isAuthorized(data) {
 }
 
 /* Popout variant */
-if (window.user_data) {
+if (user_data) {
```

Comment:
Using `window.user_data` was an attempt to make it clear its a global variable that gets injected in the global scope. It's a bit tricky because it is a variable that comes from the server and has to be sent to the client. (See the `chat.mustache` template). I do agree this should be done better, but at this point I am unsure how. Naming convention comes from the API which calls it this, but I suppose we can change that.

Reply:
I'd prefer `window.user_data` in that case so it's indeed clear where it comes from (and it keeps the linter from popping up). Perhaps it's for the best to leave the name itself as is then, to show the connection to the API.

Reply:
The only thing is that it seemed to break with `window.user_data`, that's why I reverted it again... ;-;